manifests/install.pp

```diff
@@ -63,9 +63,11 @@
         before  => File[$confluence::homedir],
         require => [
           File[$confluence::installdir],
-          User[$confluence::user],
           File[$confluence::webappdir] ],
       }
+      if ($confluence::manage_user) {
+        Staging::Extract[$file] -> User[$confluence::user]
```

Comment:
This is the wrong way round?  Should be
`User[$confluence::user] -> Staging::Extract[$file]` ?